tests/pytests/test_scorers.py

```diff
@@ -291,3 +291,19 @@ def testScoreError(env):
     waitForIndex(env, 'idx')
     env.expect('ft.add idx doc1 0.01 fields title hello').ok()
     env.expect('ft.search idx hello EXPLAINSCORE').error().contains('EXPLAINSCORE must be accompanied with WITHSCORES')
+
+def testExposeScore(env: Env):
```

Comment:
please add tests that exposes the scores of other types as well. So it'll break when something changes

Reply:
I think this is out of scope here. We only expose the existing score to the pipeline with this feature.
We should see if the existing tests don't already cover the scores of different types enough, and then add more if needed